_rules/audio-transcript-2eb176.md

```diff
@@ -22,27 +25,29 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
+This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element for which at least one of the following is true:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- the element has an `autoplay` [attribute value][] of `true`; or
+- the element has a [play button][] that is [visible][].
 
 ## Expectation
 
-The auditory information of each test target is available through a text transcript. That text transcript is [visible][] and [included in the accessibility tree][], either on the page or through a link.
+The auditory information of each test target is available through a text transcript. That text transcript is [visible][], either on the page or through a [clearly labeled location][].
 
 **Note:** A "text transcript" in the context of this rule is defined in WCAG 2 as an [alternative for time based media](https://www.w3.org/TR/WCAG22/#dfn-alternative-for-time-based-media).
 
 ## Assumptions
 
-There are no assumptions.
+Transcripts need to be available for all users, not just sighted users in order to satisfy WCAG. This rule does not test this, and it needs to be tested separately.
```

Comment:
```suggestion
In order to satisfy WCAG, transcripts need to be available for all users, not just sighted users. This rule does not test this, and it needs to be tested separately.
```
Rearranging this sentence.  My reason is that with the current arrangement, someone might think that it means this: `In order to satisfy WCAG, transcripts only need to be available to sighted users.  But WCAG doesn't go far enough - really, transcripts need to be available to all users.`   That's not true.  My suggestion clears it up.